easybuild/easyconfigs/j/Julia/Julia-1.10.2-linux-x86_64.eb

```diff
@@ -0,0 +1,31 @@
+easyblock = 'Tarball'
+
+name = 'Julia'
+version = '1.10.2'
+versionsuffix = '-linux-x86_64'
+
+homepage = 'https://julialang.org'
+description = "Julia is a high-level, high-performance dynamic programming language for numerical computing"
+
+toolchain = SYSTEM
+
+source_urls = ['https://julialang-s3.julialang.org/bin/linux/x64/%(version_major_minor)s/']
+sources = ['%(namelower)s-%(version)s%(versionsuffix)s.tar.gz']
+checksums = ['51bccc9bb245197f24e6b2394e6aa69c0dc1e41b4e300b796e17da34ef64db1e']
+
+sanity_check_paths = {
+    'files': ['bin/julia', 'include/julia/julia.h', 'lib/libjulia.%s' % SHLIB_EXT],
+    'dirs': ['bin', 'etc', 'include', 'lib', 'share']
+}
+
+sanity_check_commands = ['julia --help']
+
+modextrapaths = {
```

Comment:
this was changed from `modextrapaths_append` in `Julia-1.10.0-linux-x86_64.eb` but from the comment below it looks like appending is important?

Reply:
I may need some input here. I changed it as `modextrapaths_append` was not recognized (maybe the version did not match), but there is also a PR for this variable (#19650). I am not sure whether `modextrapaths_append` should be kept using.

Reply:
indeed, `modextrapaths_append` was only added in eb 4.9.1, you may want to upgrade eb

but yes, it's probably best to wait for the `JULIA_DEPOT_PATH` issue to settle, if nothing else because of the support for Tcl modules